packager/app/test/packager_test.py

```diff
@@ -1499,6 +1503,17 @@ def testLiveStaticProfileWithTimeInSegmentName(self):
         self._GetFlags(output_dash=True, generate_static_live_mpd=True))
     self._CheckTestResults('live-static-profile-with-time-in-segment-name')
 
+  def testAllowCodecSwitching(self):
+    streams = [
+        self._GetStream('video', test_file='bear-640x360-hevc.mp4'),
+        self._GetStream('video', test_file='bear-640x360.mp4'),
+        self._GetStream('video', test_file='bear-1280x720.mp4'),
+        self._GetStream('audio', test_file='bear-640x360.mp4'),
+    ]
+
+    self.assertPackageSuccess(streams, self._GetFlags(output_dash=True,allow_codec_switching=True))
+    self._CheckTestResults('audio-video-with-codec-switching')
```

Comment:
Unfortunately this line has to be removed as we don't generate determinstic mpd with multiple inputs right now.

Replace it with:

    # Mpd cannot be validated right now since we don't generate determinstic
    # mpd with multiple inputs due to thread racing.
    # TODO(b/73349711): Generate determinstic mpd or at least validate mpd
    #                   schema.
    # See also https://github.com/google/shaka-packager/issues/177.